src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -325,6 +325,20 @@ const policy_table::Strings& CacheManager::GetGroups(const PTString& app_id) {
   return pt_->policy_table.app_policies_section.apps[app_id].groups;
 }
 
+const policy_table::Strings CacheManager::GetPolicyAppNames() const {
```

Comment:
This should be `GetPolicyAppIDs`

Reply:
Renamed in 988d4fca0c6a9825e3e2c301905c00705bbcc901